regulation/tree.py

```diff
@@ -322,9 +322,15 @@ def build_internal_citations_layer(root):
         for cite, positions in cite_positions.items():
             # positions = find_all_occurrences(par_text, text)
             for pos in positions:
-                #print cite, positions, par_label
-                cite_dict = {'citation': cite_targets[cite],
-                             'offsets': [[pos, pos + len(cite)]]}
+                # Handle empty citations
+                try:
+                    cite_dict = {'citation': cite_targets[cite],
+                                 'offsets': [[pos, pos + len(cite)]]}
+                except TypeError as e:
+                    print "TypeError occurred: {}".format(str(e))
+                    print "Look for an empty citation in {} @ pos {}".format(par_label, positions)
```

Comment:
Only thing I'd ask is, could we `from __future__ import print_function` and use `print()` here? We've been trying to stay reasonably close to Python 3 compatibility in the XML parser (we can't go fully Python 3 because of our dependency on some bits of the eCFR parser that I'd like to rip out and library-ize some day).